couch/ci/couch.rake

```diff
@@ -1,7 +1,11 @@
 require 'ci/common'
 
 def couch_version
-  ENV['FLAVOR_VERSION'] || '1.6.1'
+  ENV['FLAVOR_VERSION'] || 'couchdb:1.6.1'
+end
+
+def couch_2?
+  couch_version.include? '2.0'
```

Comment:
While this works for now, it might not always work. Can you put a comment here saying something to that effect, we'll have to adjust it when the times comes and it'll be easier with a comment.